api/app/src/main/kotlin/packit/service/PinService.kt

```diff
@@ -20,4 +25,24 @@ class BasePinService(
             packetService.getMetadataBy(pin.packetId)
         }.sortedByDescending { it.time.start }
     }
+
+    override fun createPinByPacketId(packetId: String): Pin {
```

Comment:
Just realised since pin is for a specific packet... the DB should have a foreign key to packet? And so we can point to the packet entity and reference... is there any reason for not doing that?

Reply:
would that pin by query pin what? multiple packets? how would that work/look?

Reply:
It would refer to a specific packet, but not by id. For example, the latest packet in some group (which would change when a new packet is added to the group). https://mrc-ide.github.io/orderly2/articles/query.html